kong/runloop/balancer/ewma.lua

```diff
@@ -0,0 +1,255 @@
+--------------------------------------------------------------------------
+-- ewma balancer algorithm
+--
+-- Original Authors: Shiv Nagarajan & Scott Francis
+-- Accessed: March 12, 2018
+-- Inspiration drawn from:
+-- https://github.com/twitter/finagle/blob/1bc837c4feafc0096e43c0e98516a8e1c50c4421
+--   /finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/PeakEwma.scala
+
+
+local balancers = require "kong.runloop.balancer.balancers"
+
+local ngx_now = ngx.now
+local ngx_log = ngx.log
+local ngx_DEBUG = ngx.DEBUG
+
+local table_insert = table.insert
+
+local DECAY_TIME = 10 -- this value is in seconds
+local PICK_SET_SIZE = 2
```

Comment:
should these be configurable with sensible defaults?

Reply:
@colinh-kong I think these config no need to configurable at this moment.